cub/cub/device/dispatch/tuning/tuning_scan.cuh

```diff
@@ -203,23 +191,31 @@ struct sm80_tuning<__uint128_t, primitive_op::yes, primitive_accum::no, accum_si
 {};
 #endif
 
+template <int Threads, int Items, int L2B, int L2W>
+struct sm90_tuning_vals
+{
+  static constexpr int threads = Threads;
+  static constexpr int items   = Items;
+  using delay_constructor      = fixed_delay_constructor_t<L2B, L2W>;
+};
```

Comment:
I find it really confusing that this is not defining the same `load_algorithm` and `store_algorithm` but reuses `scan_transposed_load` from the policy hub. 

I am not sure whether this is an actual improvement

Reply:
I would be happy with a comment in the sm90 tuning that states that we intentionally use the DefaultPolicy

Reply:
I added a comment instead. I kept the new name and location for `tuning` since it only applies to SM90.